Dockerfile.template

```diff
@@ -17,7 +17,11 @@ RUN rustup target add aarch64-unknown-linux-gnu
 RUN rustup target add aarch64-unknown-linux-musl
 
 RUN curl -L https://github.com/ryankurte/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-unknown-linux-musl.tgz | tar xz -C ~/.cargo/bin
-RUN cargo binstall --no-confirm espflash
+RUN cargo binstall --no-confirm espflash --version 1.5.1
```

Comment:
1.7 is out with maximes fixes

Reply:
I just wanted to add something to the image to produce debs. But it seems like even without cargo binstall always the newest version of espflash is installed which breaks current build scripts as they suddenly require to add a CHIP parameter. 
Also I'm not really sure about binstall as it seems to fail to make installed binaries executable. I don't see the point of using it while creating the image when we could just use cargo install. On top of that we installed espflash while we actually use cargo-espflash. I need to have a closer look on Monday.
In any case we should push a new image with a new tag to not break anything. Then we can upgrade to newest espflash and fix ci scripts in a branch